### DISCUSSION TITLE
Copilot online and enabled but no suggestions appear

### DISCUSSION THREAD

**Discussion by MateuszSadowski** [zbirenbaum/copilot.lua#139] [Q&A] [answered]
I'm a Neovim newbie, so, please, bear with me. I got it briefly to work, but now I can't get any suggestions to appear even though Copilot is online and enabled. I've tried several languages. I've updated both Neovim and the plugin:
```
Neovim 0.8.3
copilot.vim 1.8.3
copilot.lua 830a679d2d8f1330caaf8c4b62e1c9a53d4cd8b8
copilot/dist/agent.js 1.73.0
Node.js 19.4.0
```
I'm using packer.nvim to install the plugin like this:
```
  use {
    "zbirenbaum/copilot.lua",
    cmd = "Copilot",
    event = "InsertEnter",
    config = function()
      require("copilot").setup({
       suggestion = {
          accept = false,
        },
      })
    end,
  }
```
and set up the tab mapping with `nvim-cmp` as described in #91.

Unfortunately, no inline or in-the-panel suggestions appear.
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/15709336/226059126-91831613-2e89-48a9-8628-367a2b8882a5.png">

How can I fix this?

**Comment by MunifTanjim** [comment, marked as answer]
Panel suggestion is not very reliable and takes a very long time to show up.

How are you trigggering inline suggestion tho? Either set `auto_trigger = true` for `suggestion`, or use the `next`/`prev` keymap to trigger manually.

Check this https://github.com/zbirenbaum/copilot.lua/issues/98#issuecomment-1363258642 for detailed guide.

**Reply by MateuszSadowski** [reply]
Thank you, @MunifTanjim! It was all about setting `auto_trigger = true` for `suggestion` for me.
